src/api-backend.js

```diff
@@ -0,0 +1,23 @@
+const postScores = async (playerName, playerScore) => {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/zeWhEaKsT4ZEAaPGExUD/scores/', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      user: playerName,
+      score: playerScore,
+    }),
+  })
+
+    .then((resp) => resp.json());
+  return response;
+};
+
+const getScores = async () => {
+  const refreshScores = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/zeWhEaKsT4ZEAaPGExUD/scores/');
+  const refreshScoresJson = refreshScores.json();
+  return refreshScoresJson;
+};
+
+export { postScores, getScores };
```

Comment:
- [ ] When I refresh to get a changed list a new list comes with all the score that was submitted before check here
![rep](https://user-images.githubusercontent.com/44745855/128518271-5a52daff-9f29-4a8a-b554-1200595f3be4.png)

Normally when you refresh it has to show an updated list with just the score you just added please fix that!

Reply:
I have resolved